uiowa/uids
# Horizontal Nav Variants

# To Test
1. Pull down this branch 
2. Change the `hidden: true` value on the second line of `src/components/menus/main-menu/main-menu.config.yml` to `hidden: false` (Do not commit this)
3. Spin up your local site with `npm run watch`
2. Under Menus > Main Menu check that all horizontal styles have:
    1. Proper `a11y` contrast for text
    2. A white active tab, unless it is `bg--white` or `bg--white--pattern--brain`, In which case they should have a Grey active tab.
    3. Yellow arrowed border at the bottom and yellow link separators, unless it is `bg--gold` or `bg--gold--pattern--brain`, in which case they should have a Grey arrowed border at the bottom and Grey link separators.
